LICENSE

```diff
@@ -0,0 +1,16 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Fastly.com
```

Comment:
we tend to put `Fastly, Inc.` here.
https://github.com/fastly/fastly_nsq/blob/master/LICENSE.txt#L1

do you mind changing it?